config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml

```diff
@@ -564,3 +564,28 @@ presubmits:
       testgrid-tab-name: capz-pr-ci-entrypoint-main
       testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
       description: Creates a CAPZ cluster and exports KUBECONFIG. This job validates ci-entrypoint.sh used by other repositories for running tests on CAPZ clusters.
+  - name: post-cluster-api-provider-azure-push-images-test
+    cluster: k8s-infra-prow-build-trusted
+    optional: true
+    always_run: false
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-image-pushes, sig-cluster-lifecycle-cluster-api-provider-azure, sig-k8s-infra-gcb
+      testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
```

Comment:
```suggestion
```

As it's a temporary job I wouldn't configure dashboards. I think we don't need it on (https://testgrid.k8s.io/) as we'll find the job runs on PRs and via prow.k8s.io